css/responsive.css

```diff
@@ -74,92 +60,175 @@
         margin-bottom: 25px;
     }
+
     .client-wrapper .col-sm-3 {
         border-right: none;
     }
-    .contact-form .col-sm-6{
+
+    .contact-form .col-sm-6 {
         margin-top: 20px;
     }
+
     .footer-3 .additional-links,
     .footer-2 .additional-links {
         clear: both;
         text-align: left;
     }
 
+    .separator-faq {
+        margin: 0 20px;
+    }
 
+    .panel-group {
+        margin-bottom: 6px;
+    }
 
+    .panel-default > .panel-heading {
+        padding: 8px 2px;
+    }
 
+    .panel-default button {
+        padding: 6px 0;
+        font-size: 16px;
+    }
 }
+
 /* Landscape phones and portrait tablets */
 @media (max-width: 767px) {
     html {
         font-size: 80%;
     }
-    .home-intro-left{
+
+    .home-intro-left {
         text-align: center;
     }
-    .home-intro-2nd-half-contact, 
-    .home-intro-2nd-half-try{
+
+    .home-intro-2nd-half-contact,
+    .home-intro-2nd-half-try {
         padding-top: 0;
     }
+
     .home-intro-2nd-half-video {
         margin-top: 50px;
     }
+
     .home-subscribe .subscribe-input {
         width: 60%;
         height: 55px;
 
     }
+
     .home-subscribe button {
         width: 30%;
         height: 55px;
         padding-left: 10px;
     }
+
     .home-button img {
         margin-bottom: 20px;
     }
+
     .service {
         margin-bottom: 30px;
     }
+
     .callnow h1 {
         font-size: 2rem;
     }
+
     .call-number {
         margin-top: 20px;
         font-size: 3rem;
         font-weight: 700;
     }
 
-    .team-member{
+    .team-member {
         width: 45%;
         float: left;
         margin: 10px;
     }
-    .contact-4 .col-sm-4,  .contact-4 .col-sm-3 {
+
+    .contact-4 .col-sm-4, .contact-4 .col-sm-3 {
         margin-top: 20px;
     }
+
     .contact-address {
-        border-top: 2px solid #EDEDED;
+        border-top: 2px solid #ededed;
         padding-top: 20px;
         margin-top: 20px;
     }
 
-    .footer-3.v-center.row > *, 
+    .footer-3.v-center.row > *,
     .footer-3 .v-center.row > * {
         display: block;
         vertical-align: baseline;
         margin-top: 30px;
     }
 
-}
+    .table-responsive {
+        border: 0;
+    }
 
+    .csstransforms .table {
+        border-collapse: collapse;
+        font-size: 12px;
+    }
+    
+    .csstransforms .table td {
+        
+    }
+
+    .csstransforms .table td:not(:first-child) {
+        width: 40px;
+    }
 
+    .csstransforms .table th {
+        height: 130px;
+        white-space: nowrap;
+    }
+
+    .csstransforms .table th > div {
+        -webkit-transform: translate(10px, 50px) rotate(-55deg);
+        -moz-transform: translate(10px, 50px) rotate(-55deg);
+        -ms-transform: translate(10px, 50px) rotate(-55deg);
+        transform: translate(10px, 50px) rotate(-55deg);
+        width: 40px;
+    }
+
+    .csstransforms .table th > div > span {
+        padding: 5px 10px;
+    }
+
+    .csstransforms .table td:not(:first-child) {
```

Comment:
You can use :not(), :fist-child, :last-child, :nth-child etc. pseudo-class selectors instead adding more classes to the HTML. https://css-tricks.com/pseudo-class-selectors/